CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/#semantic-versioning-200).
 
 
+## [1.1.5] - 2023-03-29
+### Changed
+
+* Optimized thread locks and expiring cache for the Enhanced Monitoring Plugin. ([PR #356](https://github.com/awslabs/aws-mysql-jdbc/pull/356))
+* Added checking log level before logTrace and logDebug with Messages.getString for better log consistency. ([PR #367](https://github.com/awslabs/aws-mysql-jdbc/pull/367))
+* Refactored reduce map lookups to reduce number of locks required. ([PR #373](https://github.com/awslabs/aws-mysql-jdbc/pull/373))
```

Comment:
You can add this PR to the first point `Optimized thread locks...`

Reply:
Checked in directly with Karen and we can ignore this one.